### PR TITLE
Switch to using toast instead of form alerts

### PR DESCRIPTION
**Pull request by maxime-rainville**
This PR converts some Form Alerts in asset admin to use toast notification instead. The following alerts have been converted:
- Bulk publish/unpublish/delete
- Bulk move + added a toast action to go to the folder afterwards
- Individual deletion of a file

That's all the form alerts I could find in asset-admin. 

# Parent issue
* https://github.com/silverstripe/silverstripe-admin/issues/504

# Depends on
* https://github.com/silverstripe/silverstripe-admin/pull/1068
* https://github.com/silverstripe/webpack-config/pull/48
* https://github.com/silverstripe/silverstripe-admin/pull/1125

### CODE REVIEW

File: client/src/containers/Gallery/Gallery.js [hunk 1]
```diff
@@ -572,8 +571,6 @@ class Gallery extends Component {
    */
   handleOpenFolder(event, folder) {
     event.preventDefault();
-    this.props.actions.gallery.setErrorMessage(null);
```
Comment (maxime-rainville) [inline client/src/containers/Gallery/Gallery.js:575]: This bit was testing that the message were clear. That's irrelevant when using toasts.

File: client/src/containers/MoveModal/MoveModal.js [hunk 2]
```diff
@@ -32,24 +31,18 @@ class MoveModal extends React.Component {
...
-        };
-
-        setNotice({
-          react: (
-            <span>
-              {i18n.sprintf(
-                i18n._t('AssetAdmin.MOVED_ITEMS_TO', 'Moved %s item(s) to ')
-                , selectedFiles.length
-              )}
-              <a href="#" onClick={goToFolder}>{filename}</a>
```
Comment (maxime-rainville) [inline client/src/containers/MoveModal/MoveModal.js:48]: The link will be replaced by a toast action

**Review by Cheddam — APPROVED** [review]
Looks good, ready to go once the admin PR is merged 👍

Might be a good time to bump the Composer requirements, though? Particularly the `silverstripe/admin` one?

**Comment by maxime-rainville** [comment]
Just bump the admin requirements and tweak the behat test a bit.

**Comment by maxime-rainville** [comment]
Build failure is pre-existing.

**Comment by brynwhyman** [comment]
This is minor but can we tweak the wording of these messages slightly? There's currently a combination of pluralised and non-pluralised words.

E.g we have: 
* "Moved %s **item(s)** to %s"

But we also have:
* "%s **folders/files** were successfully deleted."
* "%s **folders/files** were successfully published."

The later example looks pretty rough when there's only one file in question as it reads as "1 folders/files were successfully deleted.". Something more like the 'Moved' wording probably makes more sense.

cc @silverstripeux

File: composer.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/framework": "^4.5",
-        "silverstripe/admin": "^1.6",
+        "silverstripe/admin": "dev-pulls/1/reactstrap-goes-boom as 1.x-dev",
```
Comment (emteknetnz) [inline composer.json:8]: Needs to be reverted

File: composer.json [hunk 4]
```diff
@@ -35,5 +35,12 @@
         "process-timeout": 600
     },
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
```
Comment (emteknetnz) [inline composer.json:38]: Needs to be reverted

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Need to revert including fork before merging

File: composer.json (hunk 3 above)
Comment (emteknetnz) [reply composer.json:8]: @maxime-rainville admin pr has been merged - could you revert, push and see if travis passes
https://github.com/silverstripe/silverstripe-admin/pull/1128

Comment (maxime-rainville) [inline composer.json:8]:
```suggestion
        "silverstripe/admin": "^1.7",
```

File: composer.json [hunk 5]
```diff
@@ -35,5 +35,12 @@
...
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/open-sausages/silverstripe-admin.git",
+            "no-api": true
+        }
+    ]
```
Comment (maxime-rainville) [inline composer.json:45]:
```suggestion
    "prefer-stable": true
```

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**